src/Common/Common.Api/Workspaces/Workspace.cs

```diff
@@ -28,6 +28,19 @@ public class Workspace
 
         public string State { get; set; }
 
+        public bool IsOrphaned
+        {
+            get
+            {
+                if (this.State == null || this.State.Equals("Deleted") || this.Type.Equals("Group") || this.Type.Equals("PersonalGroup"))
```

Comment:
Is it possible to move the WorkspaceState, WorkspaceGroup, and WorkspaceUserAccessRight classes to this module so you don't have to use raw strings? I'm assuming the reason you didn't use them here is because it would create a circular dependency between the Api and Commands projects.

I think the WorkspaceUserAccessRight enum has functionality as a PowerShell parameter so please make note of that.